src/test/java/teammates/common/datatransfer/questions/FeedbackRankQuestionDetailsTest.java

```diff
@@ -25,28 +25,10 @@ public void testValidateMaxOptionsToBeRanked_sameValidValues_shouldReturnTrue()
     }
 
     @Test
-    public void testValidateMaxOptionsToBeRanked_differentValidValues_shouldReturnTrue() {
-        int testValue = 100;
-        int testDiffValue = 101;
-        FeedbackRankQuestionDetails feedbackRankQuestionDetails = new FeedbackRankOptionsQuestionDetails();
-        feedbackRankQuestionDetails.setMaxOptionsToBeRanked(testValue);
-        assertNotEquals(feedbackRankQuestionDetails.maxOptionsToBeRanked, testDiffValue);
-    }
-
-    @Test
-    public void testValidateDuplicatesAllowed_validValues_shouldReturnTrue() {
+    public void testValidateDuplicatesAllowed_isAllowed_shouldReturnTrue() {
         FeedbackRankQuestionDetails feedbackRankQuestionDetails = new FeedbackRankOptionsQuestionDetails();
         feedbackRankQuestionDetails.setAreDuplicatesAllowed(true);
         assertEquals(feedbackRankQuestionDetails.areDuplicatesAllowed, true);
```

Comment:
Sorry if I wasn't clear enough. We do want to test that a change has occured, just not test an Equals and a NotEquals of a boolean in the same state.
Something along the lines of:

- Create new object
- Test that duplicates are allowed is not true
- Set duplicates allowed to true
- Test that duplicates are allowed is now true.

We can do the same for max options as well.